python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -949,6 +982,101 @@ def calculateMetrics(self, science, difference, diaSources, kernelSources):
                 raise BadSubtractionError(ratio=metrics.differenceFootprintRatioStdev,
                                           threshold=self.config.badSubtractionVariationThreshold)
 
+    def filterSatellites(self, diaSources, science):
+
+        wcs = science.getWcs()
+        nbbox = []
+
+        for source in diaSources:
+            fp = source.getFootprint()
+            source_bbox = fp.getBBox()
+
+            corners = [wcs.pixelToSky(source_bbox.beginX, source_bbox.beginY),
+                       wcs.pixelToSky(source_bbox.beginX, source_bbox.endY),
+                       wcs.pixelToSky(source_bbox.endX, source_bbox.endY),
+                       wcs.pixelToSky(source_bbox.endX, source_bbox.beginY)]
+
+            tmp = []
+            for c, corner in enumerate(corners):
+                tmp.append([corner.getRa().asDegrees(), corner.getDec().asDegrees()])
+            nbbox.append(tmp)
+
+        detector_id = science.getDetector().getId()
+        visit_id = science.getInfo().getVisitInfo().getId()
+
+        dia_sources_json = []
+        for i, source in enumerate(diaSources):
+            dia_sources_json.append(
+                {"diasource_id": source['id'], "bbox": nbbox[i]})
+
+        sattle_output = requests.put(
+            f'{self.config.sattle_host}:{self.config.sattle_port}/diasource_allow_list',
+            json={"visit_id": visit_id, "detector_id": detector_id, "diasources": dia_sources_json})
+
+        if sattle_output.status_code != 200:
+            # Check if the cache is missing the visit and retry once
+            if sattle_output.status_code == 404:
+                try:
+                    self.log.info('Visit not found in sattle cache, re-sending')
+
+                    visit_id = science.getInfo().getVisitInfo().id
+
+                    visit_date = Time(
+                        science.getInfo().getVisitInfo().getDate().toPython()).jd
```

Comment:
```suggestion
                    visit_date = visit_info.getDate().get(DateTime.JD)
```
or
```suggestion
                    visit_date = visit_info.getDate().toAstropy().jd
```
Please don't convert via Python.